config/routes.rb

```diff
@@ -4,9 +4,13 @@
   mount Sidekiq::Web, at: '/sidekiq' if defined? Sidekiq::Web
   mount Karafka::Web::App, at: '/karafka' if ENV['KARAFKA_WEB']
 
-  mount GraphiQL::Rails::Engine, at: '/graphiql', graphql_path: '/graphql' if Rails.env.development?
+  if Rails.env.development?
+    mount GraphiQL::Rails::Engine, at: '/customer_portal_graphiql', as: 'customer_portal_graphiql_rails', graphql_path: '/customer_portal_graphql'
+    mount GraphiQL::Rails::Engine, at: '/api_graphiql', as: 'api_graphiql_rails', graphql_path: '/api_graphql'
+  end
 
-  post '/graphql', to: 'graphql#execute'
```

Comment:
You might want to keep this route temporary, otherwise the deploy will break the front-end completely :boom:

Reply:
True 👍